third-party/cmake/BuildLibvterm.cmake

```diff
@@ -67,4 +68,8 @@ BuildLibvterm(PATCH_COMMAND ${LIBVTERM_PATCH_COMMAND}
   BUILD_COMMAND ${LIBVTERM_BUILD_COMMAND}
   INSTALL_COMMAND ${LIBVTERM_INSTALL_COMMAND})
 
+ExternalProject_Add_Step(libvterm verify
+  COMMAND test -f "${DEPS_INSTALL_DIR}/lib/libvterm.a"
+  DEPENDEES install)
```

Comment:
That's the best I could find to ensure the expected output is really there, but `test` might not be there on Windows?

Also this would be something to have with other deps then also.

This showed up as problem for an IRC user, where it was using the system libvterm version, although it was built through third-party (but apparently not static).  Related: #11138.